### PR TITLE
Toolbar customization - Draw Instances

### LINKED ISSUE #614: Question: What is the correct method for adding custom buttons to Geoman?
Apologies if this is documented somewhere but I was trying to add a custom button to the geoman toolbar which participated in the button toggling that the existing buttons perform.

You can see in this fiddle that the top button toggles any other buttons which are toggled when clicked but not the other way around. Existing buttons do not toggle off the custom button when clicked.
https://jsfiddle.net/Imbick/hxt07L5j/

I may well be doing this completely wrong but couldn't find any documentation on what is the correct technique for this.

### PR DESCRIPTION
**Pull request by Falke-Design**
With this PR it is possible to add custom Controls to the Toolbar, with own actions.
Also multiple Draw Instances can be created.

Maybe it needs a few changes, but I want to talk with you about that.

Also this would be your part @codeofsumit to rewrite the sentences in the Readme, for better understanding.

Nice to know:
With that css a Toolbar with position `topleft` is automatically displayed in a new row if it goes out of the map border.
```
.leaflet-control-container > div.leaflet-top.leaflet-left {
    height: 500px;  /* map height */
    display: flex;
    flex-flow: column wrap;
}
```
But I don't know if this should be in the Readme.

Will fix #614

### CODE REVIEW

**Comment by AdamJamesNaylor** [comment]
I had to create my own version of this. Would be great if this could be merged.

**Comment by Falke-Design** [comment]
@codeofsumit tests added

File: README.md [hunk 1]
```diff
@@ -621,6 +621,96 @@ map.pm.setPathOptions({
 });
 ```
 
+You can ignore shapes when you add an array with the shape names:
+```javascript
+map.pm.setPathOptions({
+  color: 'orange',
+  fillColor: 'green',
+  fillOpacity: 0.4,
+}, ["Circle","Rectangle"]);
```
Comment (codeofsumit) [inline README.md:630]: second parameter is array of ignores shapes

File: README.md [hunk 2]
```diff
@@ -621,6 +621,96 @@ map.pm.setPathOptions({
...
+The following methods are available on `map.pm.Toolbar`:
+
+| Method                         | Returns   | Description                                                                                                   |
+| :----------------------------- | :-------- | :------------------------------------------------------------------------------------------------------------ |
+| createCustomControl(`options`) | -         | To add a custom Control to the Toolbar.                                                                       |
+| changeControlOrder(`shapes`)   | -         | Change the order of the controls in the Toolbar. You can pass all shapes and `Edit`, `Drag`, `Removal`, `Cut` |
+| getControlOrder()              | `Array`   | Get the current order of the controls.                                                                        |
+
+
+**New Draw Instance**
```
Comment (codeofsumit) [inline README.md:690]: make API easier

**Comment by Falke-Design** [comment]
@codeofsumit take a look on the new code

**Pull request merged**